bears/vcs/git/GitCommitBear.py

```diff
@@ -105,8 +105,12 @@ def run(self, allow_empty_commit_message: bool = False, **kwargs):
         stdout = stdout.rstrip('\n')
         pos = stdout.find('\n')
         shortlog = stdout[:pos] if pos != -1 else stdout
+        if pos != -1:
+            if stdout[pos+2] == '\n':
+                yield Result(self, 'Extra Newlines found.')
```

Comment:
In that case the correct position of the block was as it was before just add a `return` statement and ` if stdout[pos+1] == '\n':` instead